tests/integration/parsers/apps/nwchem/__init__.py

```diff
@@ -0,0 +1 @@
+ 
```

Comment:
I suggest keeping the `run.sh` files in git LFS and perhaps adding a comment in them, with something like:

`
# Used to re-generate test files
`

Reply:
OK, the `run.sh` scripts are now in LFS, but why did we also [move](https://github.com/Exabyte-io/express/pull/52/files/64ac3b4f7068bd2d3daec23d49606babcd055cc7..0d1e0be6dc1ab767a26e91a2eac11e6b03010b67) `references.py` there?

References we use in the tests below and we need to have them under version control.

Reply:
Not sure how they got moved. I will fix this